src/Package/MSTest.Sdk/Sdk/VSTest/VSTest.targets

```
@@ -31,7 +31,7 @@
     Ensure feature is available and user hasn't opted-out from it.
     See https://github.com/dotnet/sdk/blob/f9fdf2c7d94bc86dc443e5a9ffecbd1962b1d85d/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.CSharp.props#L26-L34
     -->
-  <ItemGroup Condition=" '$(ImplicitUsings)' == 'true' Or '$(ImplicitUsings)' == 'enable' ">
+  <ItemGroup Condition="('$(ImplicitUsings)' == 'true' Or '$(ImplicitUsings)' == 'enable') And ('$(ImplicitMSTestUsings)' == '' Or '$(ImplicitMSTestUsings)' == 'true' Or '$(ImplicitMSTestUsings)' == 'enable')">
```

I would simply go with `DisableMSTestImplicitUsings` and only check whether it's set to true.

```suggestion
  <ItemGroup Condition="('$(ImplicitUsings)' == 'true' Or '$(ImplicitUsings)' == 'enable') And '$(DisableMSTestImplicitUsings)' != 'true'">
```